src/utils/decorators/baseDecoratorExtractor.ts

```diff
@@ -2,16 +2,30 @@ import FileDetails from "../../models/fileDetails.js";
 import { ClassInfo, MethodInfo } from "./decoratorTypes.js";
 
 export abstract class DecoratorExtractor {
-    abstract getDecoratorsFromFile(file: string): Promise<ClassInfo[]>;
+    abstract getDecoratorsFromFile(file: string, cwd?: string): Promise<ClassInfo[]>;
     abstract fileFilter(cwd: string): (file: FileDetails) => boolean;
 
-    createGenezioMethodInfo(methodName: string, commentText: string): MethodInfo | undefined {
+    static createGenezioMethodInfo(
+        methodName: string,
+        commentText: string,
+    ): MethodInfo | undefined {
         const genezioMethodArguments = commentText
             .split("genezio:")[1]
             .split(" ")
             .filter((arg) => arg !== "");
+
         if (genezioMethodArguments.length < 1) {
-            return;
+            return {
+                name: methodName,
+                decorators: [
+                    {
+                        name: "GenezioMethod",
+                        arguments: {
+                            type: "jsonrpc",
+                        },
+                    },
+                ],
+            };
```

Comment:
I think this is a bit redundant because every method is exposed as jsonrpc if not explicitly marked otherwise, so we don't need to return an object here.